rrtstar_ros/bounding_box[] array